### ISSUE TITLE
About the pannuke dataset

### ISSUE THREAD

**Issue by mqbin** [vqdang/hover_net#244]
Hi, @simongraham and @vqdang , I am trying to train the pannuke dataset and I have converted the pannuke dataset to the correct format. Could you please tell me how you split the dataset for training in the paper? Did you split the three folds into a training set, a validation set, and a test set? Or did you combine these 2 as the training set and the other fold as the validation set?

and Do you use run_tile.sh to do inference on each image of the test data?
Could you tell me some specific details ？thanks！

**Comment by windygoo** [comment]
Have you solved it ?
